types/api.yaml

```diff
@@ -81,3 +81,8 @@ Committee:
       description: "List of validator indices assigned to this committee"
       items:
         $ref: './primitive.yaml#/Uint64'
+
+SyncAggregateSource:
+  type: string
+  enum: ["block", "signedcontributionandproof_pool", "synccommitteemessage_pool"]
```

Comment:
these are really hard to read...

Reply:
Thanks! Picked an awful name to trigger suggestions :smile: